src/main/config/conversion/import/iso19139.ca.HNAP.FGP-to-iso19139.ca.HNAP.xsl

```diff
@@ -102,6 +102,41 @@
     </xsl:copy>
   </xsl:template>
 
+  <!--Add default unclassified as security constraint if missing from metadata xml-->
+  <xsl:template match="gmd:MD_DataIdentification">
+    <xsl:copy copy-namespaces="no">
+      <xsl:choose>
+        <xsl:when test="gmd:resourceConstraints/gmd:MD_SecurityConstraints/gmd:classification/gmd:MD_ClassificationCode">
+        </xsl:when>
+        <xsl:otherwise>
+          <xsl:element name="gmd:resourceConstraints">
+            <xsl:element name="gmd:MD_SecurityConstraints">
+              <xsl:element name="gmd:classification">
+                <xsl:element name="gmd:MD_ClassificationCode">
+                  <xsl:attribute name="codeList">http://nap.geogratis.gc.ca/metadata/register/napMetadataRegister.xml#IC_96</xsl:attribute>
+                  <xsl:attribute name="codeListValue">RI_484</xsl:attribute>
+                  <xsl:text>unclassified; nonClassifié</xsl:text>
+                </xsl:element>
+              </xsl:element>
+            </xsl:element>
+          </xsl:element>
+        </xsl:otherwise>
+      </xsl:choose>
+      <xsl:apply-templates select="node()|@*"/>
+    </xsl:copy>
+  </xsl:template>
+
+  <!--Overwrite security constraint to default unclassified if presented in metadata xml-->
```

Comment:
Please remove comment as we are not overwriting existing classification values.  If it fails to validate user will have to change manually modify the value.

The only thing that we may overwrite is if the user note is using an old value then we may want to change it to the correct value.  Otherwise the user would have to change that as well.
See https://github.com/metadata101/iso19139.ca.HNAP/pull/195
i.e. "Protect A" could be changed to "Protected A"

Reply:
Removed.